README.md

```diff
@@ -15,14 +15,15 @@ To transform this template into an implementation, we suggest to conduct the fol
 * Place the *.ksy file of interest in the [`structs`](structs) directory
 * Update the plugin info in [`plugin.py`](plugin.py), such as:
   * the version of your plugin, author info, and your organisation
-  * the matcher on the FireFli filetype of your interest in [`plugin.py`](plugin.py) with a suitable HQL statement
-* Create test input data in the folder [`testdata/input`](testdata/input)
+  * the matcher on the `$data.fileType` property of your interest in [`plugin.py`](plugin.py) with a suitable HQL-Lite statement
+    (see also "[how to write a good matcher](https://netherlandsforensicinstitute.github.io/hansken-extraction-plugin-sdk-documentation/latest/dev/concepts/hql_lite.html#how-to-write-a-matcher)" in the documentation).
+* (optional) Add additional dependencies for your plugin to [`requirements.in`](requirements.in) if necessary 
+  * If you added additional dependencies, regenerate `requirements.txt` by calling `tox -e upgrade`
```

Comment:
did we explicitly chose to add general plugin readme to the kaitai template readme?